components/kyma-environment-broker/internal/storage/driver/postsql/events/events.go

```diff
@@ -0,0 +1,70 @@
+package events
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/storage/dbmodel"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/storage/postsql"
+	"github.com/sirupsen/logrus"
+)
+
+type Config struct {
+	Retention     time.Duration `envconfig:"default=72h"`
+	PollingPeriod time.Duration `envconfig:"default=1h"`
```

Comment:
add feature flag, enable in dev and disable in other environments

Reply:
done in https://github.com/kyma-project/control-plane/pull/2094/commits/e112e0acfdbcbab2d2daf0ee77918f9dd4a24595